### PR TITLE
Add scrapers for all "legal values"

### PR DESCRIPTION
**Pull request by JeremyKennedy**
This PR completes the automatic pulling of "legal values" from the government site. 

Instead of hardcoding various limits in a `.ts` file, now these values will be pulled with the `run-scrapers` command, along with the GIS tables. Now, when the gov updates things based on income, we should be able to reflect all that just with the `run-scrapers` command.

This change should be otherwise transparent (no functional changes).

### CODE REVIEW

File: utils/api/definitions/legalValues.ts [hunk 1]
```diff
@@ -1,22 +0,0 @@
...
- */
-export const MAX_OAS_INCOME = 133141
-export const MAX_GIS_INCOME_SINGLE = 19464
-export const MAX_GIS_INCOME_PARTNER_OAS = 25728
-export const MAX_GIS_INCOME_PARTNER_ALW = 46656 // same number as below so supposedly fine that it's unused
-export const MAX_GIS_INCOME_PARTNER_NO_OAS_NO_ALW = 46656
-export const MAX_ALW_INCOME = 36048
-export const MAX_AFS_INCOME = 26256
-
-export default MAX_OAS_ENTITLEMENT
```
Comment (KhalidAdan) [inline utils/api/definitions/legalValues.ts:22]: ❤️

Comment (JeremyKennedy) [reply utils/api/definitions/legalValues.ts:22]: A heart, but no approval? Ouch!

Comment (KhalidAdan) [reply utils/api/definitions/legalValues.ts:22]: Ahh, forgot to press the green button!

**Review by KhalidAdan — APPROVED** [review]
This looks good! ❤️ ❤️ ❤️

**Pull request merged**